packages/react/accordion/src/Accordion.tsx

```diff
@@ -61,18 +63,23 @@ type AccordionSingleOwnProps = Polymorphic.Merge<
      */
     type: 'single';
     /**
-     * The controlled stateful value of the accordion item whose panel is expanded.
+     * The controlled stateful value of the accordion item whose content is expanded.
      */
     value?: string;
     /**
-     * The value of the item whose panel is expanded when the accordion is initially rendered. Use
+     * The value of the item whose content is expanded when the accordion is initially rendered. Use
      * `defaultValue` if you do not need to control the state of an accordion.
      */
     defaultValue?: string;
     /**
      * The callback that fires when the state of the accordion changes.
      */
     onValueChange?(value: string): void;
+    /**
+     * Whether the accordion can be completely collapsed.
```

Comment:
Read this and thought if someone passes `collapsible={false}` it will still be collapsed by default which could seem like a bug given this description. Maybe we should make it more explicit?

```suggestion
     * Whether an accordion item can be collapsed after it has been opened.
```